boudicca.base/publisher-event-html/src/main/resources/templates/layout/header.hbs

```diff
@@ -139,6 +159,7 @@
                 <img src="/img/logo_small.png" alt="Boudicca Logo - Ein rundes Schwarz-auf-Weiß-Bild. Ein stilisiertes Logo der Mondkrone des keltischen Volksstammes Iceni." height="56px" width="56px"/>
                 <div>
                     <h1 class="header-title">{{title}}</h1>
+                    <h3 class="header-subtitle">All accessible events in Austria</h3>
```

Comment:
1) While I agree we should add an english version of our website soonish i think we should keep it in german for now so everything is consistent
2) While we have a focus on accessible events, those are not the only ones we have

considering those points maybe we should go with "Events in Österreich und Informationen zu deren Barrierefreiheit"?
it does not sounds as snappy as i would like but currently i have no better suggestion :D

Reply:
sure, i agree.

Reply:
![image](https://github.com/boudicca-events/boudicca.events/assets/58775404/4838c844-fbe9-4f90-9152-e6ed25c2c8c9)